charts/langwatch/values.yaml

```
@@ -134,9 +137,49 @@ langevals:
-  service:
-    type: ClusterIP
-    port: 5432
+  source: "built-in"  # "built-in" or "external"
+
+  # External PostgreSQL (when source: "external")
+  external:
+    # Reference to existing secret containing connection details
+    secretName: ""
+    secretKeys:
+      host: "host"
+      port: "port" 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)